README.md

```diff
@@ -209,7 +210,8 @@ Python plugins developers must ensure their plugin to work with all Python versi
 [csharp-example]: https://github.com/joemphilips/DotNetLightning/tree/master/examples/HelloWorldPlugin
 [csvexportpays]: https://github.com/0xB10C/c-lightning-plugin-csvexportpays
 [currencyrate]: https://github.com/lightningd/plugins/tree/master/currencyrate
-[donations]: https://github.com/lightningd/plugins/tree/master/archived/donations
+[datastore]: https://github.com/lightningd/plugins/tree/master/datastore
+[donations]: https://github.com/lightningd/plugins/tree/master/donations
```

Comment:
Same as above

Reply:
Obsolete as this change also is already in `master`.